vmdb/app/helpers/container_route_helper/textual_summary.rb

```diff
@@ -0,0 +1,45 @@
+module ContainerRouteHelper
```

Comment:
> changing this line to module ContainerProjectHelper::TextualSummary and removing next line fixes the error in my previous screenshot same for the ConatinerRoute helper. That's how we have it in other helper files.

@h-kataria , it does work for me with the current syntax.
However the reason for me doing this syntax and not the one as in the other helper files is because I got a Rubocop warning on doing ContainerProjectHelper::TextualSummary, it basically said those 2 need to be in separate lines.
I will revert to the previous syntax, but I also think that this rubocop warning needs to be turned off for future if we are not supposed to follow it anyway.